UsingAuthentication.md

```diff
@@ -101,7 +101,7 @@ public class SampleServiceController extends RestApiController {
 }
 ```
 
-Notice how the filter is setup as soon as the class is loaded, and then when a method in the sample extension called "callCore" is called, the method determines whether the call is to be tried with or without Authorization and then adds the Filter, if the Authorization is called for.  It leaves off the argument complement if not because variable arguments in methods can also handle the case when you send not arguments in.
+Notice how the filter is setup as soon as the class is loaded, and then when a method in the sample extension called "callCore" is called, the method determines whether the call is to be tried with or without Authorization and then adds the Filter, if the Authorization is called for.  It leaves off the argument complement if not because variable arguments in methods can also handle the case when you do not send arguments in.
```

Comment:
Good catch!

However, this paragraph cloud use a bit more tweaking.  Can you make these additional changes:

- Place a period after '"callCore" is called", and then start a new sentence with "The method..."

- Reword the last sentence to, "Because addAuthorization is a variable argument, the method can handle the case when you do not specify a value there".